### PR TITLE
update(FilterMenu): Use SecondaryLink for reset.

### PR DESCRIPTION
**Pull request by gaarf**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Motivation and Context

use `SecondaryLink` instead of `Link muted` for the reset link in `FilterMenu`

## Testing

styleguide

## Screenshots

before / without new prop:
Screen Shot 2020-02-06 at 1 07 49 PM

after :
Screen Shot 2020-02-06 at 3 17 39 PM

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/forms/src/components/FilterMenu/index.tsx [hunk 1]
```diff
@@ -54,6 +56,7 @@ export default function FilterMenu({
   keepOpenOnClear,
   large,
   menuProps,
+  mutedReset = true,
```
Comment (stefhatcher) [inline packages/forms/src/components/FilterMenu/index.tsx]: usually boolean value's default is `false`. please rename this to `unmuted`, default of `false` then `muted={!unmuted}`

File: packages/forms/src/components/FilterMenu/index.tsx [hunk 2]
```diff
@@ -141,7 +144,7 @@ export default function FilterMenu({
             <T k="lunar.common.apply" phrase="Apply" context="Apply filters button label" />
           </Link>
 
-          <Link muted type="reset" onClick={handleClear}>
+          <Link muted={mutedReset} type="reset" onClick={handleClear}>
```
Comment (stefhatcher) [inline packages/forms/src/components/FilterMenu/index.tsx]: is this related to a design change? it's been a standard pattern to have the clear/reset be muted

Comment (gaarf) [reply packages/forms/src/components/FilterMenu/index.tsx]: it's a want for profiles2 / sending you jira in PM

File: packages/forms/src/components/FilterMenu/index.tsx [hunk 3]
```diff
@@ -141,7 +144,7 @@ export default function FilterMenu({
             <T k="lunar.common.apply" phrase="Apply" context="Apply filters button label" />
           </Link>
 
-          <Link muted type="reset" onClick={handleClear}>
+          <Link muted={!unmutedReset} type="reset" onClick={handleClear}>
```
Comment (stefhatcher) [inline packages/forms/src/components/FilterMenu/index.tsx]: ah - chatted offline. since the concern is that the muted is too light/looks disabled (agreed), let's replace Link with SecondaryLink, remove muted. this'll make the link darker and match FormActions pattern better (primary color for main action like Submit/Apply, secondary color of Cancel/Clear). no need for a new prop then. better consistency!
